addOns/client/src/main/java/org/zaproxy/addon/client/ClientIntegrationAPI.java

```diff
@@ -41,9 +41,11 @@ public class ClientIntegrationAPI extends ApiImplementor {
 
     private static final String ACTION_REPORT_OBJECT = "reportObject";
     private static final String ACTION_REPORT_EVENT = "reportEvent";
+    private static final String ACTION_REPORT_ZEST_SCRIPT = "reportZestScript";
 
     private static final String PARAM_OBJECT_JSON = "objectJson";
     private static final String PARAM_EVENT_JSON = "eventJson";
+    private static final String PARAM_SCRIPT_JSON = "scriptJson";
```

Comment:
Same here, `statement` instead of `script`, though it could also just use `json`.

Reply:
same as above